internal/devtools.py

```diff
@@ -241,6 +243,10 @@ def start_recording(self):
         self.send_command('Debugger.enable', {})
         self.send_command('ServiceWorker.enable', {})
         self.enable_target()
+        performance_metrics_script = os.path.join(self.script_dir, 'performance_metrics.js')
+        if os.path.isfile(performance_metrics_script):
+            with io.open(performance_metrics_script, 'r', encoding='utf-8') as script_file:
+                self.send_command('Page.addScriptToEvaluateOnNewDocument', {'source': script_file.read()})
```

Comment:
Reading the docs for `Page.addScriptToEvaluateOnNewDocument` it appears the script will get evaluated in every frame that gets created?

https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-addScriptToEvaluateOnNewDocument